lib/src/youtube_clone/HomePage.dart

```diff
@@ -0,0 +1,200 @@
+/**
+ * Author: Aparna Dulal
+ * profile: https://github.com/Ambikadulal
+  */
+import 'package:flutter/material.dart';
+
+class HomePage extends StatefulWidget {
+  static const String path = 'lib/src/youtube_clone/HomePage.dart';
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  Widget eachvideo(String assets, String title) {
+    return Column(
+      children: <Widget>[
+        Stack(
+          children: <Widget>[
+            Image.asset(assets),
+            Positioned.fill(
+              bottom: 10.0,
+              right: 10.0,
+              child: Align(
+                alignment: Alignment.bottomRight,
+                child: Container(
+                  color: Colors.white,
+                  padding: EdgeInsets.all(4.0),
+                  child: Text(
+                    "20:10",
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+        SizedBox(
+          height: 5.0,
+        ),
+        Container(
+          color: Colors.black,
+          child: ListTile(
+            leading: CircleAvatar(
+              backgroundImage: NetworkImage(
+                  'https://cdn.logo.com/hotlink-ok/logo-social.png'),
```

Comment:
You can use links variables from assets.dart instead of hardcoding. 
Use other images not logos..